tests/correctness/methods.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2020 Ghaith Hachem and Mathias Rieder
+use super::super::*;
+
+#[test]
+fn fb_vars_can_be_accessed() {
+    let src = "
+FUNCTION main : DINT 
+    prg();
+    main := prg.y;
+END_FUNCTION
+
+PROGRAM prg
+    VAR x : myFB; END_VAR
+    VAR_OUTPUT y : DINT; END_VAR
+    y := x.test(32);
+END_PROGRAM
+
+FUNCTION_BLOCK myFB
+VAR
+    x : INT := 10;
+END_VAR
+METHOD test : DINT

```

Comment:
I think you meant:
```
METHOD test : DINT
   VAR_INPUT a : DINT; END_VAR
   test := x + a;
END_METHOD
```

these tests dont work even when I fixed the ordering ... so I changed them to `test := x + a` which is what you probably were aiming for